src/main/java/net/minestom/server/registry/Registry.java

```diff
@@ -0,0 +1,52 @@
+package net.minestom.server.registry;
+
+import net.minestom.server.attribute.Attribute;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.metadata.villager.VillagerProfession;
+import net.minestom.server.entity.metadata.villager.VillagerType;
+import net.minestom.server.fluid.Fluid;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockEntity;
+import net.minestom.server.instance.block.BlockState;
+import net.minestom.server.item.Enchantment;
+import net.minestom.server.item.Material;
+import net.minestom.server.particle.Particle;
+import net.minestom.server.potion.PotionEffect;
+import net.minestom.server.potion.PotionType;
+import net.minestom.server.sound.SoundEvent;
+import net.minestom.server.statistic.StatisticType;
+
+/**
+ * AUTOGENERATED
+ */
+public final class Registry {
+    public static final IdCrossMapRegistry.Defaulted<Block> BLOCK_REGISTRY = new IdCrossMapRegistry.Defaulted<>(() -> Block.AIR);
```

Comment:
These don't need to end with registry. I think `Registry.BLOCKS` would be enough for example.